backend/src/controllers/boards/edit-board.js

```diff
@@ -0,0 +1,26 @@
+import { connection as knex } from '../../database/connection.js';
+import format from 'date-fns/format';
+
+const editBoard = async (req, res)=>{
+	const { title, favorited } = req.body;
+	const id = req.params.id;
+	try{
+		const boardEdit = await knex('boards').update({
+			title,
+            favorited,
+            user_id: 2,
+            creation_date: format(new Date(), 'yyyy-MM-dd kk:mm:ss'),
```

Comment:
O creation_date não entra na função de update. Quando for fazer a edição do board, só mudamos o update_date pq estamos atualizando ele de fato... mas não mudamos o creation_date, que é fixo sempre pq marca quando o board foi criado.